jbpm-flow/src/main/java/org/jbpm/ruleflow/instance/RuleFlowProcessInstance.java

```diff
@@ -34,9 +34,11 @@ public RuleFlowProcess getRuleFlowProcess() {
 
     public void internalStart(String trigger) {
     	StartNode startNode = getRuleFlowProcess().getStart(trigger);
-    	if (startNode != null) {
-    		((NodeInstance) getNodeInstance(startNode)).trigger(null, null);
-    	}
+        if (startNode != null) {
+            ((NodeInstance) getNodeInstance(startNode)).trigger(null, null);
+        } else if (!getRuleFlowProcess().isDynamic()) {
+            throw new IllegalArgumentException("There is no start node that matches the trigger " + trigger == null ? "none" : trigger);
```

Comment:
@elguardian There are still parentheses missing around the ternary operator, string concatenation has higher priority.
So it should be:
IllegalArgumentException("There is no start node that matches the trigger " + (trigger == null ? "none" : trigger));

In other words, 
`"There is no start node that matches the trigger " + trigger == null` is always false.